data_structures/queue_with_stacks.py

```diff
@@ -0,0 +1,34 @@
+from stack.stack import Stack
+
+
+class Queue_with_stacks:
```

Comment:
Class names with more than one word are in PascalCase. Thus this should be `QueueWithStacks`.